test/dataset/test_feature_transforms.py

```diff
@@ -0,0 +1,42 @@
+import tempfile
+import pytest
+import torch
+
+from lhotse import CutSet
+from lhotse.dataset.feature_transforms import GlobalMVN
+
+
+@pytest.fixture
+def global_mvn():
+    cuts = CutSet.from_json('test/fixtures/ljspeech/cuts.json')
+    return GlobalMVN.from_cuts(cuts)
+
+
+def test_global_mvn_initialization_and_stats_saving(global_mvn):
+
+    with tempfile.NamedTemporaryFile() as tf:
+        global_mvn.to_file(tf.name)
+        global_mvn2 = GlobalMVN.from_file(tf.name)
+
+    for key_item_1, key_item_2 in zip(
+            global_mvn.state_dict().items(),
+            global_mvn2.state_dict().items()
+    ):
+        assert torch.equal(key_item_1[1], key_item_2[1])
+
+
+@pytest.mark.parametrize(
+    "in_tensor", [torch.ones(10, 40), torch.ones(2, 10, 40)]
+)
+def test_global_mvn_shapes(global_mvn, in_tensor):
+    assert global_mvn(in_tensor).shape == in_tensor.shape
+    assert global_mvn.inverse(in_tensor).shape == in_tensor.shape
+
+
+@pytest.mark.parametrize(
+    "in_tensor", [torch.ones(10, 40), torch.ones(2, 10, 40)]
```

Comment:
I'm not sure I understand why it works with both of these shapes. Could you explain?

Reply:
I guess its torch broadcasting. Try
```
(torch.randn(5) - torch.randn(1, 2, 5)).shape
(torch.randn(5) - torch.randn(2, 5)).shape
```

Reply:
Or should it not work?

Reply:
OK I'll check it out later. Merging!

Reply:
NP. I tested it now and it returns the same result even when I unsqueeze the smaller tensor.